frontend/components/ProjectCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="project-card" @click="handleClick">
+    <div class="project-name">
+      <span class="icon">🚀</span>
+      {{ name }}
+    </div>
+    <div class="name-underline"></div>
+    <ul>
+      <li v-for="technology in technologies" :key="technology" class="technology-item">{{ "• " + technology }}</li>
+    </ul>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
Prefer using the [Composition API](https://vuejs.org/guide/typescript/composition-api.html) rather than the [Options API](https://vuejs.org/guide/typescript/options-api.html) as it is more reusable and to be consistent with the other components

Reply:
Done